common/src/main/java/org/conscrypt/Conscrypt.java

```diff
@@ -67,6 +67,34 @@ public static X509TrustManager getDefaultX509TrustManager() throws KeyManagement
     }
 
     /**
+     * Utility that exposes common TLS constants.
+     */
+    public static final class Constants {
+        private Constants() {}
+
+        /**
+         * Returns the maximum length (in bytes) of plaintext data.
+         */
+        public static int maxPlaintextLength() {
+            return NativeConstants.SSL3_RT_MAX_PLAIN_LENGTH;
```

Comment:
I mean, this constant seems mostly fine. I'm just trying to figure out what you're trying to do and giving you something you may wish to think about.

If the API is to answer "how much buffer to I need to be able to fit a decrypted record", then this constant is solid. If it's to answer "how much may I be guaranteed to fit in a single record for this particular socket", you could imagine situations changing.